### DISCUSSION TITLE
Erro 403 forbidden na autenticação com o Bradesco.

### DISCUSSION THREAD

**Discussion by Netow** [bacen/pix-api#344] [General]
Alguém que já realizou a integração com o Bradesco e poderia dar uma ajuda sobre o erro 403 forbidden, muito provavelmente relacionado a autenticação com o certificado digital???
Já realizei a integração com o Itaú de forma rápida e sem transtornos.

**Comment by rubenskuhl** [comment]
Não é por causa daqueles headers proprietários onde o Bradesco desobedece o regulamento do BACEN, como os X-Brad-* ?

**Comment by Netow** [comment]
Então, na documentação que eu tenho, não menciona esses headers para obtenção do token. No caso, eu não estou passando nenhum header deste tipo. Apenas Basic authoraztion , o grant_type e o certificado

**Comment by rubenskuhl** [comment]
> Então, na documentação que eu tenho, não menciona esses headers para obtenção do token. No caso, eu não estou passando nenhum header deste tipo. Apenas Basic authoraztion , o grant_type e o certificado

Talvez falar com o suporte.api@bradesco.com.br para ver se a documentação que você tem reflete o que está em produção.

**Reply by Netow** [reply]
Fiz um teste pelo postman passando os mesmos header, parametros e certificado e funcionou. 
O Bradesco disse que a ferramenta deles pra suporte é o postman. Entendi como se fosse um "se vira aí"
Em tempo, a api ja roda com o Itaú, que em termos de implementação não tem diferenca. O que muda sao as credencias de cada banco.

**Comment by EduardoDosSantosPereira** [comment]
Boa tarde. 

Estou tendo este problema com as API's de produção do Itau, quando consumo elas via POSTMAN informando o certificado .crt e a chave privada funciona. Porem usando via cURL no PHP eu obtenho o seguinte retorno de HTTP/1.1 401 Unauthorized:

{
"message" : "Falha ao Extrair TokenInfo filter failed"
}

Não sei se sua integração é feita com PHP. 
Se puder me ajudar eu agradeço.

Obrigado!
